### DISCUSSION TITLE
i am getting error message. Can you help me?

### DISCUSSION THREAD

**Discussion by idldsgn** [Abasz/ESPRowingMonitor#7] [Q&A]
I tried to compile the project and it fails.
c:/users/mint works/.platformio/packages/toolchain-xtensa-esp32s3/bin/../lib/gcc/xtensa-esp32s3-elf/8.4.0/../../../../xtensa-esp32s3-elf/bin/ld.exe: cannot open map file C:/Users/MINT WORKS/Desktop/[안전폴더]ESPRowingMonitor-master/ESPRowingMonitor-master/.pio/build/generic/firmware.map: Invalid argument
collect2.exe: error: ld returned 1 exit status
*** [.pio\build\generic\firmware.elf] Error 1

Can you heplme?

**Comment by idldsgn** [comment]
I will buy FireBeetle 2 ESP32-E IoT Microcontroller and do it again

**Reply by Abasz** [reply]
This seems to me a tooling issue not a board issue. You need to setup platformio correctly and then you need to set platformio.ini file correctly too.

In the platformio.ini first and for most you need to set the correct board. Assuming you want to use an ESP32 S3 bard you need to change this:
`board = firebeetle32`
to the name of the board you have (e.g. for the lollin s3 mini):
`board = lolin_s3_mini`

In addition, I found an github issue: https://github.com/platformio/platform-espressif32/issues/1068, which suggests to have only ASCII characters in the path.

**Comment by idldsgn** [comment]
Thanks. I've succeeded to upload the program to my board. I changed board name and ram size.
How can I connect with  web gui?

**Reply by Abasz** [reply]
> How can I connect with web gui?

Via bluetooth as described in the README: https://github.com/Abasz/ESPRowingMonitor-WebGUI

**Reply by Abasz** [reply]
Were you able to set gui and the ergo up?

**Comment by idldsgn** [comment]
Were you able to set gui and the ergo up?

**Reply by Abasz** [reply]
sorry? :)

**Reply by idldsgn** [reply]
So Sorry. I replied thru phone app but something went wrong!

Since I am not good at coding, I am going to visit one of my friend who is expert.  I tried my best but  was not successful.

**Comment by idldsgn** [comment]
sorry? :)
